curriculum/challenges/english/blocks/workshop-word-counter/6921d90b5ce50185505d2004.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 6921d90b5ce50185505d2004
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Before adding a loop to the function, first implement a minimal version of its functionality.
```

Comment:
I vote we remove this sentence all together. 

The goal of this step seems clear enough with that last sentence.